doc-build/third_party/hf-doc-builder/src/doc_builder/autodoc.py

```diff
@@ -197,7 +197,7 @@ def regex_closure(object_doc, regex):
     object_doc = remove_example_tags(object_doc)
     object_doc = hashlink_example_codeblock(object_doc, anchor)
 
-    svelte_str = "<div>\n"
+    svelte_str = "<div style={{width: \"200px\", height: \"200px\", overflow: \"auto\"}}>\n"
```

Comment:
Will the div be extremly big after removing this style?

Reply:
I just want to use "overflow",  When the content exceeds the box, a scroll bar will be generated